<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="22.743" tests="2" failures="1">
  <testsuite name="Root Suite" timestamp="2025-09-30T17:58:48" tests="0" file="cypress\e2e\simpleLoginTests.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="login tests" timestamp="2025-09-30T17:58:48" tests="2" time="22.740" failures="1">
    <testcase name="login tests successful login" time="0.000" classname="successful login">
      <failure message="Timed out retrying after 4000ms: expected &apos;&lt;a#nameofuser.nav-link&gt;&apos; to have text &apos;Welcome admin&apos;, but the text was &apos;&apos;" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: expected '<a#nameofuser.nav-link>' to have text 'Welcome admin', but the text was ''
    at Context.eval (webpack://demoblaze/./cypress/page_objects/loginPage.js:17:48)
    at getRet (https://demoblaze.com/__cypress/runner/cypress_runner.js:121618:20)
    at tryCatcher (https://demoblaze.com/__cypress/runner/cypress_runner.js:1777:23)
    at Promise.attempt.Promise.try (https://demoblaze.com/__cypress/runner/cypress_runner.js:4285:29)
    at Context.thenFn (https://demoblaze.com/__cypress/runner/cypress_runner.js:121629:66)
    at Context.then (https://demoblaze.com/__cypress/runner/cypress_runner.js:121880:21)
    at wrapped (https://demoblaze.com/__cypress/runner/cypress_runner.js:141596:19)

      + expected - actual

      +'Welcome admin'
      ]]></failure>
    </testcase>
    <testcase name="login tests [negative] non-existing user login attempt" time="8.175" classname="[negative] non-existing user login attempt">
    </testcase>
  </testsuite>
</testsuites>